UnitOfWork/Models/BaseEntity.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace UnitOfWork.Models;
+
+public class BaseEntity
+{
+    public int Id { get; set; }
+    public DateTime CreatedAt { get; set; }
+    public DateTime UpdatedAt { get; set; }
+}
```

Comment:
Add 2 more fields: `string CreatedBy` and nullable `string? UpdatedBy`.

Reply:
added